java/semantickernel-api/src/main/java/com/microsoft/semantickernel/DefaultKernel.java

```diff
@@ -40,15 +40,15 @@ public DefaultKernel(
     }
 
     @Override
-    public <T> Mono<ContextVariable<T>> invokeAsync(
+    public <T> Mono<FunctionResult<T>> invokeAsync(
```

Comment:
Maybe this would be a good time to move DefaultKernel implementation to Kernel and do away with DefaultKernel.